gslib/command_runner.py

```diff
@@ -137,6 +137,18 @@ def _StringToSysArgType(unicode_str):
   return unicode_str
 
 
+def _OriginalPrint(s=''):
```

Comment:
I ended up removing `_OriginalPrint` later because I was able to use `ttyprint` instead with less breakages. Any `_originalPrint` commits I made can be ignored safely

Reply:
ok, I will remove